src/estimators/directive/index.ts

```diff
@@ -26,6 +21,10 @@ export default function (options?: {}): ComplexityEstimator {
       }
     },
   });
+}
+
+export default function (options: { name?: string } = {}): ComplexityEstimator {
```

Comment:
Let's create a type for this that we can then also export `ComplexityDirectiveOptions` and reuse externally.